typescript/infra/scripts/safes/create-safe.ts

```diff
@@ -0,0 +1,87 @@
+import { SafeFactory } from '@safe-global/protocol-kit';
+import { SafeAccountConfig } from '@safe-global/protocol-kit';
+
+import { getChainMetadata } from '../../config/registry.js';
+import {
+  GCP_PROJECT_ID,
+  getSecretManagerServiceClient,
+} from '../../src/utils/gcloud.js';
+import { readJSONAtPath } from '../../src/utils/utils.js';
+import {
+  getArgs,
+  withChainRequired,
+  withSafeTxServiceUrlRequired,
+  withThreshold,
+} from '../agent-utils.js';
+
+const OWNERS_FILE_PATH = 'config/environments/mainnet3/safe/safeSigners.json';
+
+async function main() {
+  const { chain, safeTxServiceUrl, threshold } = await withThreshold(
+    withSafeTxServiceUrlRequired(withChainRequired(getArgs())),
+  ).argv;
+
+  const chainMetadata = await getChainMetadata();
```

Comment:
could maybe use `getChain(chain)` (also from registry.ts) as it looks like we just use a single chain